packages/terra-framework-site/themeable-variables.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
I'd probably set this as an empty object to start with `{}`. Once the plugin is run it will update this file. I'm also fine if we leave as is and let the plugin update this file later.